contrib/datawave-quickstart/bin/services/accumulo/bootstrap.sh

```diff
@@ -57,7 +66,9 @@ DW_ACCUMULO_VFS_DATAWAVE_DIR="/datawave/accumulo-vfs-classpath"
 # accumulo.properties (Format: <property-name>=<property-value>{<newline>})
 
 DW_ACCUMULO_PROPERTIES="## Sets location in HDFS where Accumulo will store data
-instance.volumes=${DW_HADOOP_DFS_URI}/accumulo
+instance.volumes=${DW_HADOOP_DFS_URI_CLIENT}/accumulo
+
+tserver.memory.maps.native.enabled=false
```

Comment:
@keith-ratcliffe is this ok?  This has been a hangup for me a few times now when loading the quickstart on my local bo (i.e. not in docker).  I added this for testing purposes but didn't intend to commit it.  What are your thoughts?